### PR TITLE
Bug 1713181 Updating registry operator docs

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1713181

### CODE REVIEW

File: modules/registry-operator-config-resources.adoc [hunk 1]
```diff
@@ -45,6 +45,16 @@ data:
...
+. To configure an additional CA:
++
+----
+$ oc create configmap registry-config --from-file=<cert> -n openshift-config
+$ oc edit image.config.openshift.io cluster
+spec:
+  additionalTrustedCA:
+    name: registry-config
+----
+
```
Comment (wzheng1) [inline modules/registry-operator-config-resources.adoc]: It is better to add how to check the change succeed:

Then you can check inside image-registry pod:
$ oc rsh image-registry-xxxxx
sh-4.2$ ls /etc/pki/ca-trust/source/anchors
<external_registry_address>  image-registry.openshift-image-registry.svc..5000  image-registry.openshift-image-registry.svc.cluster.local..5000

File: modules/registry-operator-config-resources.adoc [hunk 2]
```diff
@@ -45,6 +45,16 @@ data:
 <1>  If the registry has the port, such as `registry-with-port.example.com:5000`.
 : should be replaced with `..`.
 
+. To configure an additional CA:
++
+----
+$ oc create configmap registry-config --from-file=<cert> -n openshift-config
```
Comment (wzheng1) [inline modules/registry-operator-config-resources.adoc]: Should add external registry address after --from-file, like
$oc create configmap registry-config --from-file=<external_registry_address>=ca.crt -n openshift-config

**Comment by bmcelvee** [comment]
Thanks so much, @wzheng1!

**Comment by wzheng1** [comment]
LGTM, thanks!

**Comment by vikram-redhat** [comment]
Received confirmation from QE. Merging.

**Pull request merged**